src/ModelForm/ModelFormActions.tsx

```diff
@@ -40,7 +40,9 @@ export const ModelFormActions = ({
   };
   const onSaveHandler = handleSubmit(async (formData: DataType) => {
     const changedData = Object.fromEntries(
-      Object.entries(formData).filter((entry) => dirtyFields[entry[0]]),
+      Object.entries(formData)
+        .filter((entry) => dirtyFields[entry[0]])
+        .map((entry) => [entry[0], entry[1] === '' ? null : entry[1]]),
```

Comment:
Why did u add this? If a user puts an empty string, it must stay an empty string

Reply:
I can see this being a problem if on magql, the dev does not specify that a field is of type `[str | None]`. Other than that, I think an absence of a value should be treated as null.